harvdev_utils/production/production.py

```diff
@@ -257,6 +257,10 @@ def primary_id(self):
         """Fetch primary_id."""
         return self.cell_line_id
 
+    def first_id(self):
```

Comment:
Docs at top say that "first_id" is only for join tables like CellLineCvterm, etc. Seems like this should not be added.

Reply:
This has been Remo 3rd, I found a better way.